src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs

```diff
@@ -297,8 +297,24 @@ public override void Dispose()
             }
         }
 
-        // Transforms the web.config file to include the hostingModel="inprocess" element
-        // and adds the server type = Microsoft.AspNetServer.IIS such that Kestrel isn't added again in ServerTests
+        private void ModifyDotNetExePathInWebConfig()
+        {
+            // We assume the x64 dotnet.exe is on the path so we need to provide an absolute path for x86 scenarios.
```

Comment:
Let's not assume, let's always set the full path to dotnet we think is correct to run.